collector/src/main/java/com/navercorp/pinpoint/collector/service/TraceService.java

```diff
@@ -74,6 +74,10 @@ public void insertSpan(final SpanBo spanBo) {
         insertSpanEventStat(spanBo);
     }
 
+    private void insertAcceptorHost(SpanEventBo spanEvent, String applicationId, ServiceType serviceType) {
+        hostApplicationMapDao.insert(spanEvent.getEndPoint(), spanEvent.getDestinationId(), spanEvent.getServiceType(), applicationId, serviceType.getCode());
```

Comment:
Is it necessary? since there are going to be checked inside `HbaseHostApplicationMapDao.insert()` method.

Reply:
The check inside `HbaseHostApplicationMapDao.insert()` throws a **NullPointerException**, so it will skip all span event statistics that come after this from being stored.

Reply:
taken care of